db/migrate/20160406142950_fix_versions_item_type_for_steps.rb

```diff
@@ -0,0 +1,9 @@
+class FixVersionsItemTypeForSteps < ActiveRecord::Migration
+  def up
+    execute "UPDATE versions SET item_type = 'Steps::Approval' WHERE item_type = 'Approval'"
+  end
+
+  def down
+    execute "UPDATE versions SET item_type = 'Approval' WHERE item_type = 'Steps::Approval'"
```

Comment:
we don't want any `versions` rows with `item_type='Approval'` because there is no such class as `Approval`. The date is just for archeological purposes.